### PR TITLE
Danielle baxter

### PR DESCRIPTION
**Pull request by danibaxx**

### CODE REVIEW

File: components/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -2,27 +2,29 @@ class Dropdown {
...
+    // console.log(this.element);
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = this.element.querySelector('.dropdown-button');
+      // console.log(this.button);
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
+    this.content = document.querySelector('.dropdown-content');
```
Comment (ari-mercado) [inline components/Dropdown/Dropdown.js:12]: this.button = this.element.querySelector('.dropdown-button');

File: components/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -2,27 +2,29 @@ class Dropdown {
...
+
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
+    this.button.addEventListener('click', () => this.toggleContent()); {
```
Comment (ari-mercado) [inline components/Dropdown/Dropdown.js:16]: The curly braces after the semicolon are not needed.

File: components/Tabs/Tabs.js [hunk 3]
```diff
@@ -59,4 +68,6 @@ class TabItem {
 
 */
 
-links = document.querySelectorAll();
\ No newline at end of file
+links = document.querySelectorAll('.tabs-link').forEach(link => {
```
Comment (ari-mercado) [inline components/Tabs/Tabs.js:71]: links variable can be declared using let or const.

File: components/Tabs/Tabs.js [hunk 4]
```diff
@@ -2,50 +2,59 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = document.querySelector(`.tabs-link[data-tab='${this.element.dataset.tab}']`);
```
Comment (ari-mercado) [inline components/Tabs/Tabs.js:8]: Another way to get the custom data attribute is by using
   * this.data = this.element.dataset.tab;*

**Review by ari7946 — APPROVED** [review]
## Code Review Checklist

- [x] Follows Lambda Git Flow 
- [x] Frequent, clear, and descriptive commits
- [x] No errors in the console
- [x] Code is written by the student
- [x] Code is sufficiently commented
- [x] Code is organized and easy to read
- [x] MVP Complete
- [ ] At least 1 stretch completed

### Notes:
Nice job, Danielle.

**Pull request merged**